doc/source/whatsnew/v0.21.0.txt

```diff
@@ -99,6 +99,7 @@ Indexing
 ^^^^^^^^
 
 - When called with a null slice (e.g. ``df.iloc[:]``), the``iloc`` and ``loc`` indexers return a shallow copy of the original object. Previously they returned the original object. (:issue:`13873`).
+- Fixed a bug that prevented joining on a categorical MultiIndex (:issue:`13873`).
```

Comment:
Bug in joining on a ``MultiIndex`` with a ``Categorical`` level. Issue number is incorrect